jinja_utils_test.py

```diff
@@ -108,3 +119,17 @@ def test_evaluate_object(self):
         parsed_dict = jinja_utils.evaluate_object(orig_dict, {'b': 'c'})
         self.assertEqual(orig_dict, {'a': '{{b}}'})
         self.assertEqual(parsed_dict, {'a': 'c'})
+
+        # int type input used.
+        parsed_object = jinja_utils.evaluate_object(34, {})
+        self.assertEqual(parsed_object, 34)
+
+    def test__log2_floor_filter(self):
+        # disabling pylint warning for protected access
```

Comment:
I don't think you need this comment now that the human readable pylint code is used.

Reply:
Removed it.!